app/controllers/step_by_step_pages_controller.rb

```diff
@@ -101,6 +101,8 @@ def unpublish
     end
   end
 
+  def unschedule; end
```

Comment:
Thank you for doing this! 🎉 It's great that you added a placeholder so the tests pass, but that all of the functionality isn't added in one giant commit, which makes it easier to review.